### PR TITLE
Update free-programming-books-subjects.md

### PR DESCRIPTION
**Pull request by atharva20-coder**
## What does this PR do?
Add resource(s)

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by atharva20-coder** [comment]
Made required changes as stated on #8704 issue @shrn01

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -708,6 +710,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Professional Software Development For Students](https://mixmastamyk.bitbucket.io/pro_soft_dev/intro.html) - Mike G. Miller
 * [Software Engineering at Google](https://abseil.io/resources/swe-book) - Titus Winters, Tom Manshreck, Hyrum Wright
 * [Software Environment Concepts](https://softwareconcepts.vercel.app) - Amr Elmohamady (:construction: *in process*)
+* [Programming-with-JS](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
```
Comment (shrn01) [inline books/free-programming-books-subjects.md:713]: This will go into the languages file

Comment (shrn01) [inline books/free-programming-books-subjects.md:713]: This will go into the languages file

Comment (atharva20-coder) [reply books/free-programming-books-subjects.md:713]: Okay

**Pull request closed without merging**